lib/CapabilityDelegation.js

```diff
@@ -38,10 +38,15 @@ module.exports = class CapabilityDelegation extends ControllerProofPurpose {
    *   the creation of the proof.
    * @param [maxTimestampDelta] {integer} a maximum number of seconds that
    *   the date on the signature can deviate from, defaults to `Infinity`.
+   * @param {function} [inspectCapabilityChain] - An async function used to
+   *  check the capability chain. It can be use to find revocations related
+   *  to any of the capabilities in the chain. The expected return value
+   *  is {valid: true/false, error}. The function is sent {capabilityIds}
```

Comment:
Expected return value is a Promise that resolves to that object, right?

Reply:
It's described as an async function, is that not adequate?

Reply:
I don't know if it's adequate or not. If you think it is, then it is until someone files an issue complaining that it isn't (that won't be me).